context.go

```diff
@@ -71,6 +71,11 @@ func (c *Context) Config() (*Config, error) {
 		return nil, fmt.Errorf("Unknown error in GetConfig")
 	}
 
+	// Set finalizer for free C pointer on gc
```

Comment:
It seems get config from `NewConfig` and rely on `SetFinalizer` in there
	
```
config, err := NewConfig()
if err != nil {
   return nil, err
}
```

passes CI

Reply:
Good catch, done